.travis.yml

```diff
@@ -8,10 +8,33 @@ php:
   - 7.0
   - hhvm
 
+matrix:
+    allow_failures:
+        - php: hhvm
+        - php: 5.3
+          env: SYMFONY_VERSION=2.1.*
+    include:
+        - php: 5.3
+          env: SYMFONY_VERSION=2.1.*
+        - php: 5.3
+          env: SYMFONY_VERSION=2.3.*
+        - php: 5.6
+          env: SYMFONY_VERSION=2.7.*
+        - php: 5.6
+          env: SYMFONY_VERSION=2.8.*
+        - php: 5.6
+          env: SYMFONY_VERSION=3.0.*
+        - php: 7.0
+          env: SYMFONY_VERSION=3.0.*
+
+env:
+    - SYMFONY_VERSION=2.3.*
+
 before_install:
   - travis_retry composer self-update
 
 install:
+  - composer require symfony/framework-bundle:${SYMFONY_VERSION}
   - composer install --ignore-platform-reqs
```

Comment:
We can also think about it like this, and introduce the `COMPOSER_FLAGS` too:

https://github.com/liip/LiipHyphenatorBundle/blob/master/.travis.yml#L33